proposals/nnnn-navigation-subscription-buttons.md

```diff
@@ -0,0 +1,46 @@
+# Navigation Subscription Buttons
+
+* Proposal: [SDL-NNNN](NNNN-navigation-subscription-buttons.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / Cloud Proxy / HMI / RPC]
```

Comment:
```suggestion
* Impacted Platforms: [Core / iOS / Java Suite / HMI / RPC]
```

As both the cloud proxy and android library now exist under `Java Suite` we'd like to use that as a platform name moving forward, thanks!